SCClassLibrary/Common/GUI/PlusGUI/Control/ServerPlusGUI.sc

```diff
@@ -160,14 +167,24 @@ ServerTreeView {
 			}
 		}, '/g_queryTree.reply', this.server.addr).fix;
 
-		updateFunc = {
-			updater = fork {
+		updater = {
+			fork {
 				loop {
+					defer {
+						if (this.server.serverIsRunning) {
+							treeViewStatus.string_(" STARTED: current server tree");
+						} {
+							treeViewStatus.string_(" STOPPED: last updated server tree")
+						}
+					};
 					this.server.sendMsg("/g_queryTree", 0, 0);
 					interval.wait;
 				}
 			}
 		};
+		updateFunc = {
```

Comment:
you can write `updater = updateFunc`

Reply:
I'm sorry, but I didn't understand it. The two options aren't working.
```
updater = updateFuncl
```
and
```
updater = updateFunc {
updater.value
}:

Reply:
two functions `updateFunc = { someFunction.value }` is equivalent to `updateFunc = someFunction` (if the function doesn't take arguments)

Reply:
Ah, ok, you mean
`updateFunc = updater` It works.